modules/fluid_properties/include/userobjects/NaKFluidProperties.h

```diff
@@ -0,0 +1,86 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "SinglePhaseFluidProperties.h"
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Woverloaded-virtual"
+
+/**
+ * NaK fluid properties as a function of pressure (Pa) and temperature (K).
+ *
+ * Origin of fluid properties:
+ * 1) density; thermal diffusivity; dynamic viscosity; specific heat capacity
```

Comment:
I don't know what this means. Does it mean this object computes these properties or requires them as inputs?

Reply:
no it means these fluids properties comes from this book

Reply:
if we have more properties in the future they can come from the same book or another book, just list it there in the header